### PR TITLE
Publish the CLI plugin for ndc-postgres v0.4.0.

### PR DESCRIPTION
**Pull request by SamirTalwar**

### CODE REVIEW

File: plugins/ndc-postgres/20240304/manifest.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+name: ndc-postgres
+version: "v0.4.0"
```
Comment (SandeepSamba) [inline plugins/ndc-postgres/20240304/manifest.yaml:2]: The version here should match the file path. So either both should be `20240304` or `v0.4.0`.

File: plugins/ndc-postgres/20240304/manifest.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+shortDescription: "CLI plugin for Hasura ndc-postgres"
+homepage: https://hasura.io/connectors/postgres
+platforms:
+  - selector: darwin-arm64
+    uri: "https://github.com/hasura/ndc-postgres/releases/download/v0.4.0/ndc-postgres-cli-aarch64-apple-darwin"
+    sha256: "478fa4b81ad13666a9546b6d58355a37737babd0bc96c3fd49583be88a3d498a"
```
Comment (SandeepSamba) [inline plugins/ndc-postgres/20240304/manifest.yaml:8]: Each platform object should contain the `bin` property (which here should have the value `hasura-ndc-postgres`) and the `files` property which should have a `from` and `to` properties. Sample file:

```yaml
name: ndc-postgres
version: "v0.4.0"
shortDescription: "CLI plugin for Hasura ndc-postgres"
homepage: https://hasura.io/connectors/postgres
platforms:
  - selector: darwin-arm64
    uri: "https://github.com/hasura/ndc-postgres/releases/download/v0.4.0/ndc-postgres-cli-aarch64-apple-darwin"
    sha256: "478fa4b81ad13666a9546b6d58355a37737babd0bc96c3fd49583be88a3d498a"
    bin: hasura-ndc-postgres
    files:
      - from: ./ndc-postgres-cli-aarch64-apple-darwin
        to: hasura-ndc-postgres
  - selector: linux-arm64
    uri: "https://github.com/hasura/ndc-postgres/releases/download/v0.4.0/ndc-postgres-cli-aarch64-unknown-linux-gnu"
    sha256: "3284b5466d8101df3ed03f44d56a4eb006f0e7d892c3ed613d9a56a3e992e801"
    bin: hasura-ndc-postgres
    files:
      - from: ./ndc-postgres-cli-aarch64-unknown-linux-gnu
        to: hasura-ndc-postgres
  - selector: darwin-amd64
    uri: "https://github.com/hasura/ndc-postgres/releases/download/v0.4.0/ndc-postgres-cli-x86_64-apple-darwin"
    sha256: "8c51023f7d6233109b96a00899491771683eb401be455787543989f7fa8b616a"
    bin: hasura-ndc-postgres
    files:
      - from: ./ndc-postgres-cli-x86_64-apple-darwin
        to: hasura-ndc-postgres
  - selector: windows-amd64
    uri: "https://github.com/hasura/ndc-postgres/releases/download/v0.4.0/ndc-postgres-cli-x86_64-pc-windows-msvc.exe"
    sha256: "34e738903eafb6faa5f51e7b94494caaecd9c45e0246157a6ed193ce0a9eb712"
    bin: hasura-ndc-postgres
    files:
      - from: ./ndc-postgres-cli-x86_64-pc-windows-msvc.exe
        to: hasura-ndc-postgres.exe
  - selector: linux-amd64
    uri: "https://github.com/hasura/ndc-postgres/releases/download/v0.4.0/ndc-postgres-cli-x86_64-unknown-linux-gnu"
    sha256: "6009b1f7157ffad8a5a05958c6f390871b2a98d4af087886c84d5f4cbb0cd942"
    bin: hasura-ndc-postgres
    files:
      - from: ./ndc-postgres-cli-x86_64-unknown-linux-gnu
        to: hasura-ndc-postgres

```

**Review by SandeepSamba — CHANGES_REQUESTED** [review]
Please make the suggested changes in the structure of the plugin manifest

File: plugins/ndc-postgres/20240304/manifest.yaml (hunk 2 above)
Comment (SamirTalwar) [reply plugins/ndc-postgres/20240304/manifest.yaml]: Done, thanks!

**Pull request merged**